piet-cairo/src/text.rs

```diff
@@ -123,8 +208,31 @@ impl TextLayoutBuilder for CairoTextLayoutBuilder {
         self
     }
 
-    fn alignment(self, _alignment: piet::TextAlignment) -> Self {
-        // TextAlignment is not supported by cairo toy text.
+    fn alignment(self, alignment: TextAlignment) -> Self {
+        //TODO: This will not respect Start/End of right-to-left scripts
```

Comment:
Ayyy, that's great news! I don't quite like how it causes Left/Right to be swapped in interpretation instead of having them be called Start/End but I can live with that. I'll add a note here to explain that behavior